conf/messages

```diff
@@ -18,6 +18,7 @@ global.opening.times.8to7_30=Monday to Friday, 8am to 7:30pm
 global.opening.times.8to4.includesat=Monday to Saturday, 8am to 4pm
 global.opening.times.mon.sat.8to7_30=Monday to Saturday, 8am to 7:30pm
 global.opening.times.saturday.8to4=Saturday, 8am to 4pm
+global.opening.times.saturday.8to330=Saturday, 8am to 3:30pm
```

Comment:
for consistency across the messages file can put _ for 3:30 PM
"global.opening.times.saturday.8to3_30"

Reply:
taking this out and creating child benefits own Saturday opening hours key as it has it's own view (not global)